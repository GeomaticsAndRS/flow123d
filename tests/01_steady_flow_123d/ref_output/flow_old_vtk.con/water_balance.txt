	"time"	"region"	"quantity"	"flux"	"flux_in"	"flux_out"	"mass"	"source"	"source_in"	"source_out"	"integrated_flux"	"integrated_source"	"error"
	0	"region_62"	"water_volume"	0	0	0	0	0	0	0	0	0	0
	0	"region_63"	"water_volume"	0	0	0	0	0	0	0	0	0	0
	0	"region_64"	"water_volume"	0	0	0	0	0	0	0	0	0	0
	0	"region_65"	"water_volume"	0	0	0	0	0	0	0	0	0	0
	0	"IMPLICIT BOUNDARY"	"water_volume"	-3.58642e-13	-2.2107	2.2107	0	0	0	0	0	0	0
