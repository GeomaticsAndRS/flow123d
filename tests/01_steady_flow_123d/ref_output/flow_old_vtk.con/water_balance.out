********************************
Boundary fluxes at time inf:
[total balance]    [total outflow]     [total inflow]
boundary id:-3 label:'IMPLICIT BOUNDARY'	-2.40086e-11	2.2107	-2.2107
